sceptre/synapseprod/config/prod/snowflake-access.yaml

```diff
@@ -0,0 +1,6 @@
+template:
+  path: snowflake-access.yaml
+stack_name: snowflake-accesss
+parameters:
+  SnowflakeAccountArn: !ssm /infra/SnowflakeAccountArn
```

Comment:
I'm not as knowledgable about this, but where are these values pulled from?

Reply:
Agree about account, will actually make things easier as we first have to setup the role with fake trust relationship values, then update with actual values.